src/test/java/io/github/thebusybiscuit/slimefun4/testing/tests/tasks/TestArmorTask.java

```diff
@@ -73,7 +73,9 @@ void testSlimefunArmor() throws InterruptedException {
     @DisplayName("Test Radiation and Hazmat Suits")
     @MethodSource("cartesianBooleans")
     void testRadiactivity(boolean hazmat, boolean radioactiveFire) throws InterruptedException {
-        Player player = server.addPlayer();
```

Comment:
commented out stuff...Just a reminder so this does not stay in the PR

Reply:
Okay. I will keep the reminder until the PR is at the merging stage